vagrant/src/main/vagrant/servers.yaml

```diff
@@ -41,7 +41,9 @@ servers:
     box: ubuntu/wily64
     ram: 2048
     cpus: 4
-    ip: 10.10.10.100
+    forwarded_ports:
+     - guest: 8081
+       host: 8081
```

Comment:
If someone has another process listening on port 8081, then presumably `vagrant up brooklyn` will fail with a nice message? Is that ok to assume they won't have anything else running there? I think yes.